Scripts/DCS-BIOS/lib/common/JSONHelper.lua

```diff
@@ -0,0 +1,47 @@
+module("JSONHelper", package.seeall)
+
+--- @class JSONHelper
+local JSONHelper = {}
+
+--- @func Encodes to JSON and writes to file
+--- @param value any
+--- @param filename string File name including path
+function JSONHelper.encode_to_file(value, filename)
+	local JSON = BIOS.json
+	local file, err = io.open(filename, "w")
+
+	if(err ~= nil)then
+		error("Failed to open file "..filename.." for writing => "..err, 1)
+	end
+
+	local json_string = JSON:encode_pretty(value)
+	if file then
+		file:write(json_string)
+		file:close()
+		file = nil
+	end
+end
+
+--- @func Encodes to JSON and writes to file
+--- @param filename string File name including path
+--- @return any
```

Comment:
return type should be specified (it's a `table`, I believe? `{ [string]: any }` maybe?)